gazelle/python/BUILD.bazel

```
@@ -53,13 +54,19 @@ filegroup(
     output_group = "python_zip_file",
 )
 
-go_test(
+gazelle_test(
     name = "python_test",
     srcs = ["python_test.go"],
     data = [
         ":gazelle_binary",
         ":helper",
-    ] + glob(["testdata/**"]),
+    ],
+    testdata = glob(
+        # Use this so that we don't need to manually maintain the list.
+        ["testdata/*"],
+        exclude = ["testdata/*.md"],
+        exclude_directories = 0,
+    ),
```

@rickeylev, is this an appropriate use for a glob?

---

Oh I see. The directories are being included because they're then used to feed another glob to create per-directory targets. I was going to comment that directories shouldn't be direct inputs, but that isn't the case here. Just a small comment to clarify. Renaming testdata to test_dirs would also help clarify.

```suggestion
    testdata = glob(
        # Use this so that we don't need to manually maintain the list.
        ["testdata/*"],
        exclude = ["testdata/*.md"],
        # The directories aren't inputs themselves; we just want their names.
        exclude_directories = 0,
    ),
```